src/node/blockstorage.h

```diff
@@ -302,6 +306,7 @@ class BlockManager
 
 public:
     using Options = kernel::BlockManagerOpts;
+    using ReadRawBlockResult = util::Expected<std::vector<std::byte>, ReadRawError>;
```

Comment:
nit in d2ffbb5985717f9b6348a7aab60fcad6e271a296: This is only used twice. Could remove it, or move it right above the function declaration  that needs it?